nsxt/provider.go

```diff
@@ -437,6 +438,8 @@ func Provider() *schema.Provider {
 			"nsxt_edge_high_availability_profile":          resourceNsxtEdgeHighAvailabilityProfile(),
 			"nsxt_policy_host_transport_node_collection":   resourceNsxtPolicyHostTransportNodeCollection(),
 			"nsxt_policy_lb_client_ssl_profile":            resourceNsxtPolicyLBClientSslProfile(),
+			"nsxt_policy_security_policy_rule":             resourceNsxtPolicySecurityPolicyRule(),
+			"nsxt_policy_security_policy_no_rule":          resourceNsxtPolicySecurityPolicyNoRule(),
```

Comment:
I feel that `nsxt_policy_security_policy_no_rule` is confusion, especially next to `nsxt_policy_security_policy_rule`.
I don't have a much better option, but lets not give up on better naming yet :)
`nsxt_policy_empty_security_policy`? `nsxt_policy_parent_security_policy`?
There is also an option to remove the first `policy` part, since non-policy resources are deprecated by now.
This would result in `nsxt_security_policy` and `nsxt_security_policy_rule`.

Reply:
Changed to nsxt_policy_parent_security_policy. Ideas are welcome!